allennlp/interpret/attackers/hotflip.py

```diff
@@ -156,7 +156,8 @@ def attack_from_json(self,
                 current_instance_labeled = self.predictor.predictions_to_labeled_instances(current_instance,
                                                                                            outputs)[0]
                 # if the prediction has changed, then stop
-                if any(current_instance_labeled[field] != fields_to_compare[field] for field in fields_to_compare):
+                if any(not current_instance_labeled[field].__eq__(fields_to_compare[field])
```

Comment:
Don't call `.__eq__`, just use `==`.  I only did that in my example to force using the base class implementation.

Reply:
Oh, you shouldn't need to change this line at all.  Why do you need to change this?  Changing the other parts should have been sufficient.

Reply:
O I see. I didn't know how equals works in python.